internal/job/vault-cred-sync.go

```diff
@@ -121,6 +121,11 @@ func (v *VaultCredSync) Run() {
 			v.log.Infof("credentail type %s not supported", key)
 		}
 	}
+	// After successfully syncing data to Vault, delete the Kubernetes Secret.
+
+	if err = k8s.DeleteKubernetesSecret(ctx, v.conf.VaultCredSyncSecretName, v.conf.VaultSecretNameSpace); err != nil {
```

Comment:
it doesn't confirm here, vault store successful.
in above error logged and continued.
have to check all passed in above for loop